gnuradio-runtime/lib/block.cc

```diff
@@ -47,12 +47,12 @@ block::block(const std::string& name,
       d_pmt_done(pmt::intern("done")),
       d_system_port(pmt::intern("system"))
 {
-    global_block_registry.register_primitive(alias(), this);
+    global_block_registry.register_primitive(registry_name(), this);
     message_port_register_in(d_system_port);
     set_msg_handler(d_system_port, boost::bind(&block::system_handler, this, _1));
 }
 
-block::~block() { global_block_registry.unregister_primitive(symbol_name()); }
```

Comment:
@jdemel something i did not understand about the existing codebase is why `alias()` is registered but `symbol_name()` is unregistered. Do you understand this? I think these changes in the PR are no longer necessary but wanted to understand this a little better.